src/types.d.ts

```diff
@@ -9,74 +25,83 @@ interface Options {
    * The default log level. Winston will filter messages with a severity lower
    * than this.
    */
-  level?: string;
```

Comment:
Why are there changes from `;` to `,`?  I would have expected the change to go the other way.

Reply:
Checked trace agent and debug agent we use ';', I have reverted it back.